internal/nostr/nostr.go

```diff
@@ -757,14 +775,23 @@ func (svc *Service) startSubscription(ctx context.Context, subscription *Subscri
 			svc.Logger.WithFields(logrus.Fields{
 				"subscription_id": subscription.ID,
 				"relay_url":       subscription.RelayUrl,
-				"wallet_pubkey":   svc.getWalletPubkey(subscription.Authors),
+				"wallet_pubkey":   walletPubkey,
+				"client_pubkey":   clientPubkey,
 			}).Info("Stopping subscription")
 			break
 		}
 	}
 }
 
 func (svc *Service) publishEvent(ctx context.Context, subscription *Subscription) {
+	walletPubkey := ""
+	clientPubkey := ""
+
+	if (subscription.RequestEvent != nil) {
+		walletPubkey = svc.getWalletPubkey(subscription.Authors)
```

Comment:
why is it not taken from the `subscription.RequestEvent` here?

Reply:
No specific reason, I thought it's easier to get from Authors array, but on second thought I think using nostr.Tag methods should mke it easier, too `subscription.RequestEvent.Tags.GetFirst([]string{"p", ""}).Value()`